src/gateway/understanding-kong/how-to/request-transformations.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Adding attributes to HTTP requests with Kong Gateway"
+description: "A no-code solution to modifying requests for your APIs"
+---
+
+It's very common to have an HTTP service which accepts requests expecting a JSON document in the body.
+Let's assume that the development team of your service plans to change the request API in the near future, 
+and will eventually begin to require a new field in the JSON body of the requests. 
+Eventually your client applications will need to upgrade their requests to support this new value, 
+but how could you provide a default value for your services in the meantime?
+
+[{{site.base_gateway}}](/gateway/{{page.kong_version}}/) supports a [Plugin](/hub/) 
+architecture including a [Request Transformer Plugin](/hub/kong-inc/request-transformer/) 
+that can modify incoming requests before proxying them to your upstream service. 
+This can all be accomplished using a no-code solution and managed with no downtime using 
+Kong's dynamic administrative capabilities.
+
+This guide will show you how to configure the Request Transformer plugin using 
+the [Kong Admin API](/gateway/{{page.kong_version}}/admin-api/) to modify incoming 
+requests with a static constant value. Then you will test the feature with a mock 
+request to verify the transformation process.
+
+### Prerequisites
+
+The guide assumes the following:
+
+* You have a {{site.base_gateway}} instance available for testing with connection information
+stored in the following environment variables:
+    * `KONG_PROXY` contains the URL to the {{site.base_gateway}} proxy endpoint
+    * `KONG_ADMIN_API` contains the URL to the {{site.base_gateway}} admin API endpoint
+
+  If you would like to quickly setup a local {{site.base_gateway}} using Docker and setting up these variables, 
+use this companion [{{site.base_gateway}} in minutes](/gateway/{{page.kong_version}}/how-to/kong-gateway/) guide. 
+If your gateway is available on a different host and port, adjust the commands in this guide appropriately.
+* You have a service named `mock` installed on your gateway. You can adjust the 
+commands to use a different service or use the [{{site.base_gateway}} in minutes](/gateway/{{page.kong_version}}/how-to/kong-gateway) guide to 
+run a gateway with a `mock` service.
```

Comment:
```suggestion
  If you would like to quickly setup a local {{site.base_gateway}} using Docker and setting up these variables, 
use this companion [{{site.base_gateway}} in minutes](/gateway/{{page.kong_version}}/how-to/kong-gateway/) guide. 
If your gateway is available on a different host and port, adjust the commands in this guide appropriately.
* You have a service named `mock` installed on your gateway. You can adjust the 
commands to use a different service or use the [{{site.base_gateway}} in minutes](/gateway/{{page.kong_version}}/how-to/kong-gateway) guide to 
run a gateway with a `mock` service.
```

I think we should just condense both of these points into a single bullet that says assumes the user followed the previous doc.

Reply:
Please see revised version